client/views/room/contextualBar/Call/Jitsi/CallJitsiWithData.js

```diff
@@ -182,7 +182,7 @@ const CallJitsiWithData = ({ rid }) => {
 
 			jitsi.on('HEARTBEAT', testAndHandleTimeout);
 
-			shouldDispose = jitsi.openNewWindow;
+			shouldDispose = !jitsi.openNewWindow;
```

Comment:
```suggestion
			shouldDispose = jitsi.openNewWindow;
```
With this change, calls are not working when the "Always open in new window" setting is off (since the call would be disposed). I think we could keep it as it was and your fix would still work well, right?

Reply:
can you tell me what should be the behaviour ? In which state should we dispose the call and in which we should not ?
As per the suggested changes if `shouldDispose = jitsi.openNewWindow`  then 
```
fetchData().then(() => {
			if (shouldDispose) {
				jitsi.dispose();
			}
		});
```

this always runs and dispose the call as soon as it starts when the call is opened in the new window.